gitpod-io/gitpod
# Demo PR: Example of using the new with-integration-tests checkbox

## Description


This is just a demo PR to provide an example of how you can use the new with-integration-tests checkbox.

## Related Issue(s)


Just a demo.

## How to test


Check the checkboxes and `/werft run` a job

![Screenshot 2022-09-19 at 12 05 28](https://user-images.githubusercontent.com/83561/190995406-5cab25e7-aaf6-4778-b02b-9eead4b572ac.png)

In this case the integration tests failed with "no suitable user found: make sure there's at least one non-builtin user in the database (e.g. login)" but that's a known requirement which is documented in [the README](https://github.com/gitpod-io/gitpod/tree/main/test).

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

- [x] /werft with-integration-tests=webapp
      Valid options are `all`, `workspace`, `webapp`, `ide`
